api/routing/handler.go

```diff
@@ -104,3 +119,14 @@ func (response *Response) writeTo(w http.ResponseWriter) error {
 
 	return nil
 }
+
+func (response *Response) yamlEncoder(w http.ResponseWriter) error {
```

Comment:
similar to the function above, I would recommend calling the function `encodeToYaml`